amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/index.tsx

```diff
@@ -108,7 +108,7 @@ class DetailListItem extends React.Component<DetailListItemProps, DetailListItem
               value={metadata.description}
             />
           </div>
-          <Dropdown pullRight={true}>
+          <Dropdown id='detail-list-item-dropdown' pullRight={true}>
```

Comment:
We render multiple `DetailListItem` components so I'd imagine there might be some error somewhere regarding duplicate ids. Luckily the column index happens to be available on this component, for now we can append `:${this.props.index}` to the id.